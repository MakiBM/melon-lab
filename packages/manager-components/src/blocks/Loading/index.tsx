import React from 'react';

import styles from './styles.css';

const Loading = () => {
  return (
    <span className="loading">
      <style jsx>{styles}</style>
      <span className="loading__item">.</span>
      <span className="loading__item">.</span>
      <span className="loading__item">.</span>
    </span>
  );
};

export default Loading;
