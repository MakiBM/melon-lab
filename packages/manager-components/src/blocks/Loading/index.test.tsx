import React from 'react';
import Loading from './index';

describe('Button', () => {
  const defaultElement = <Loading />;

  it('should render correctly', () => {
    const wrapper = shallow(defaultElement);
    expect(wrapper).toMatchSnapshot();
  });
});
