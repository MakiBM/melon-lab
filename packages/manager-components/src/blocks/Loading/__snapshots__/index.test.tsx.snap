// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button should render correctly 1`] = `
<React.Fragment>
  <JSXStyle
    css="__scoped"
    styleId="__scopedHash"
  />
  <span
    className="jsx-__scopedHash loading__item"
  >
    .
  </span>
  <span
    className="jsx-__scopedHash loading__item"
  >
    .
  </span>
  <span
    className="jsx-__scopedHash loading__item"
  >
    .
  </span>
</React.Fragment>
`;
